kubernetes/ftest-kubernetes/src/test/java/HelloWorldTest.java

```diff
@@ -0,0 +1,28 @@
+import io.fabric8.kubernetes.api.model.Service;
+import org.arquillian.cube.kubernetes.annotations.Named;
+import org.arquillian.cube.kubernetes.impl.requirement.RequiresKubernetes;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.io.IOException;
+
+import static org.junit.Assert.*;
+
+@RunWith(Arquillian.class)
+@RequiresKubernetes
+public class HelloWorldTest {
+
+    @Named("hello-world-service")
+    @ArquillianResource
+    Service helloWorld;
+
+    @Test
+    public void shouldShowHelloWorld() throws IOException {
+        assertNotNull(helloWorld);
+        assertNotNull(helloWorld.getSpec());
+        assertNotNull(helloWorld.getSpec().getPorts());
+        assertFalse(helloWorld.getSpec().getPorts().isEmpty());
```

Comment:
Looks awesome to me 💃 
One question if we want to use fabric8 assertion mechanism, should be possible as well by just adding the dependency right?